packages/vue-components/src/panels/NestedPanel.vue

```diff
@@ -268,6 +268,7 @@ export default {
     }
 
     .button-wrapper {
+        display: inline-block;
```

Comment:
I am seeing a warning about the use of `display:inline-block` together with `float:right`, wonder if this is ok?
![image](https://user-images.githubusercontent.com/41845017/183879688-fdd94ca9-1486-4bf0-9ce9-8e05ea0f266c.png)
https://developer.mozilla.org/en-US/docs/Web/CSS/float

Reply:
Yup seems like `display: inline-block` will override the `float: right` property for minimal panels. `float: right` is still needed for the buttons on the normal panels.